src/model/cloud-runner/workflows/ephemeral-github-runner-workflow.ts

```diff
@@ -0,0 +1,43 @@
+import CloudRunnerEnvironmentVariable from '../services/cloud-runner-environment-variable';
+import CloudRunnerLogger from '../services/cloud-runner-logger';
+import CloudRunnerSecret from '../services/cloud-runner-secret';
+import { CloudRunnerState } from '../state/cloud-runner-state';
+import { CloudRunnerStepState } from '../state/cloud-runner-step-state';
+import { WorkflowInterface } from './workflow-interface';
+
+export class EphemeralGitHubRunnerWorkflow implements WorkflowInterface {
+  async run(cloudRunnerStepState: CloudRunnerStepState) {
+    try {
+      await EphemeralGitHubRunnerWorkflow.runJobAsEphemeralGitHubRunner(
+        cloudRunnerStepState.image,
+        cloudRunnerStepState.environment,
+        cloudRunnerStepState.secrets,
+      );
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  private static async runJobAsEphemeralGitHubRunner(
+    image: string,
+    environmentVariables: CloudRunnerEnvironmentVariable[],
+    secrets: CloudRunnerSecret[],
+  ) {
+    try {
+      CloudRunnerLogger.log(`Cloud Runner is running in ephemeral GitHub runner mode`);
+      const installAndStartRunner =
+        ' cd .. & cd .. && ls && mkdir actions-runner && cd actions-runner && curl -O -L https://github.com/actions/runner/releases/download/v2.283.1/actions-runner-linux-x64-2.283.1.tar.gz && tar xzf ./actions-runner-linux-x64-2.283.1.tar.gz';
```

Comment:
you can also `cd ../../`

Reply:
This entire workflow needs a bit of cleanup. This was just some basic exploring, hence the pointless repitition.

Reply:
Gonna save on a development branch and just delete this workflow for this PR